Remotion/Security/UnitTests/XmlAsserter/XmlElementConstraint.cs

```diff
@@ -25,26 +25,27 @@ public class XmlElementConstraint : Constraint  {
     private readonly string _expectedNamespace;
     private readonly string _expectedLocalName;
 
+    public override string Description => "Test";

```

Comment:
PairConstraint doesn't have the override of Description. Why? Also "Test" feels like it's a leftover bit? So, delete here?

Reply:
the description was previously written in the `WriteDescriptionTo` method; This method was overridden in xmlconstraint but not in pairconstraint.
Since it seems like a placeholder/dummy text i will remove it.

Reply:
Ah there it is. yes, please do.